packages/batch-submitter/src/exec/run-batch-submitter.ts

```diff
@@ -18,12 +16,13 @@ import {
 } from '..'
 
 /* Logger */
-const destination = createWriteStream({
-  dsn: process.env.SENTRY_DSN,
-  tracesSampleRate: 0.05,
-  level: 'error'
+const log = new Logger({
+  name: 'oe:batch-submitter:init',
```

Comment:
Is this conforming to some known format?

Reply:
i have no idea... @karlfloersch might have more context here